src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -580,6 +614,137 @@ export class HaAutomationEditor extends PreventUnsavedMixin(
     }
   }
 
+  public connectedCallback() {
+    super.connectedCallback();
+    window.addEventListener("paste", this._handlePaste);
```

Comment:
We probably want to move all paste logic to the `manual-automation-editor`, as it is only used in that editor.

Now if you paste an automation config while in a blueprint automation, you are switched to a manual automation with the config (and an append gets even weirder in a blueprint automation ofc)